pkg/reconciler/events/pubsub/pubsub.go

```diff
@@ -37,25 +37,13 @@ import (
 )
 
 const (
-	finalizerName = controllerAgentName
-
-	resourceGroup = "cloudpubsubsources.events.cloud.google.com"
+	creatingFailedReason    = "PullSubscriptionGetFailed"
```

Comment:
nit: can you group the reasons separately from finalizerName and resourceGroup.
Say something like:

```
const(
     finalizerName = 
     resourceGroup = 

     createFailedReason
     other reason....
)```

Reply:
also, your creating reason is talking about Get... you will need a new reason

Reply:
done